lighthouse-core/audits/byte-efficiency/unused-css-rules.js

```diff
@@ -29,7 +29,7 @@ class UnusedCSSRules extends Audit {
     return {
       category: 'CSS',
       name: 'unused-css-rules',
-      description: 'Avoids loading unnecessary CSS',
+      description: 'Unused CSS Rules',
```

Comment:
Haha, yeah might be worth having a dedicated report messaging discussion just to iron out what we want to standardize on :)